framework/platform/bpl/uci/db/bpl_db_uci.cpp

```diff
@@ -144,6 +144,69 @@ bool uci_set_section(const std::string &package_name, const std::string &section
     LOG(TRACE) << "uci_set_section() " << package_name << ".(" << section_type << ")"
                << section_name;
 
+    auto ctx = alloc_context();
+    if (!ctx) {
+        return false;
+    }
+
+    for (auto &option : options) {
+        char opt_path[MAX_UCI_BUF_LEN] = {0};
+        if (snprintf(opt_path, MAX_UCI_BUF_LEN, option_path, package_name.c_str(),
+                     section_name.c_str(), option.first.c_str()) <= 0) {
+            LOG(ERROR) << "Failed to compose path";
+            return false;
+        }
+
+        LOG(TRACE) << "Trying to lookup: " << opt_path;
```

Comment:
```suggestion
        LOG(DEBUG) << "Trying to lookup: " << opt_path;
```
in general - if you go into rework of the PR - make sure the logging is as we discussed

Reply:
Removed the unneeded logs throughout the code